compiler/testData/codegen/bytecodeText/inlineClasses/resultApiDoesntCallSpecializedEquals.kt

```diff
@@ -0,0 +1,13 @@
+// WITH_COROUTINES
```

Comment:
This is really minor, but maybe these tests will succeed even with `WITH_RUNTIME`, now that `Result` in the standard library.

Reply:
Indeed they do, fixed!